index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet" />
+  <title>Kartikay's Portfolio</title>
+</head>
+
+<body class="main-block">
+  <header class="head-menu">
+    <nav class="nav-mobile">
+      <a class="logo" href="#">My Logo</a>
+      <i class="fa fa-bars"></i>
+    </nav>
+  </header>
+  <section class="home-page">
+    <div class="home-page-wrapper">
+      <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
+      <!-- <h1 class="intro">Glad to see you!</h1> -->
+      <p class="intro-description">
+        I’m a software developer! I can help you build a product, feature or
+        website. Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact
+        me.
+      </p>
+      <h3 class="bottom-connect">LET’S CONNECT</h3>
```

Comment:
As you already named the class, we can figure out that it should be a button.

- [ ] Please replace the `h3` tag with an `a` or `button` tag.

Reply:
Thanks for the suggestion. I have done the required changes.